src/World.js

```diff
@@ -3,9 +3,11 @@ import { EntityManager } from "./EntityManager.js";
 import { ComponentManager } from "./ComponentManager.js";
 import { Version } from "./Version.js";
 import { hasWindow, now } from "./Utils.js";
+import { Entity } from "./Entity.js";
 
 const DEFAULT_OPTIONS = {
```

Comment:
`DEFAULT_OPTIONS`, is modified in the `World` constructor can you change it to `this.options = Object.assign({}, DEFAULT_OPTIONS, options)`

Reply:
nice catch, fixed